README.md

```diff
@@ -102,8 +102,47 @@ fn fetch_an_integer() -> String {
 }
 ```
 
+## JSON Support
+
+Support for the RedisJSON Module can be enabled by specifying "json" as a feature in your Cargo.toml.
+
+`redis = { version = "0.17.0", features = ["json"] }`
+
+Then you can simply include the `json::Commands` which will add the commands to all Redis Connections
+
+```rust
+use redis::Client;
+use redis::json::Commands;
+use redis::RedisResult;
+
+// Result returns Ok(true) if the value was set
+// Result returns Err(e) if there was an error with the server itself 
+fn set_json_bool<P: ToString>(key:P, path:P, b: bool) -> RedisResult<bool> {
+    let client = Client::open("redis://127.0.0.1").unwrap();
+    let connection = client.get_connection().unwrap();
+
+    // runs `JSON.SET {key} {path} {b}`
+    connection.json_set(key, path, b)?
+}
+
+```
+
+**NOTE:** Importing both `redis::Commands` and `redis::json::Commands` may cause you build to fail, 
```

Comment:
Can we export `redis::json::Commands`, etc., from the commands module with different names so that we don't have the potentially confusing build failures?

Reply:
In this case `redis::Commands` and `redis::json::Commands` are implementations of the same trait. I'm not certain if the trait could be renamed but I can test or perhaps I can figure something out.

the `redis::json::Commands` is imported conditionally in the `lib.rs` file https://github.com/d3rpp/redis-rs-tokio-drift/blob/da865eef905e67437c78afdf05427130899cff77/redis/src/lib.rs#L443

I could *attempt* to make it export as `redis::JsonCommands` for example but I can guarantee nothing.

Reply:
that was surprisingly easy